doc/api.apib

```diff
@@ -20,7 +20,7 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 
 # Group Workflows
 
-## Workflow [/v1/workflows/{component}/{workflow_id}]
```

Comment:
Unsure if removing v1 altogether is correct, given that it's not sunset yet. Should it be kept in a separate file?